src/components/pages/import-batch-view.yml

```diff
@@ -19,15 +19,9 @@ taskStatusFailed:
 taskStatusDoneWithWarnings:
   en: Done (with warnings)
   de: Abgeschlossen (mit Warnungen)
-createdOn:
-  en: Created on
-  de: Erstellt am
 completedOn:
```

Comment:
Maybe this is something that could go into `common` too, what do you think? It's very general.

Reply:
True, it's used on 2 import pages.
Extracted.